ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -151,8 +158,12 @@ private static boolean isManualExecution() {
 
     private static void createTableAndMapping() {
         try {
-            queryClient.executeToJsonResult(DB_NAME, String.format(".drop table %s ifexists", tableName));
+            queryClient.executeToJsonResult(DB_NAME, String.format(".drop table %s ifexists ", tableName));
             queryClient.executeToJsonResult(DB_NAME, String.format(".create table %s %s", tableName, tableColumns));
```

Comment:
Maybe out of scope here: do we want to check the results?

Reply:
Out of scope
I really dont undersantd why we started checking the results elsewhere - did engine started to return valid results on failures?